src/common/actions/repositories.js

```diff
@@ -1,18 +1,20 @@
 import 'isomorphic-fetch';
+import { push } from 'react-router-redux';
 
 import { checkStatus, getError } from '../helpers/api';
 import { conf } from '../helpers/config';
+import { fetchUserSnaps } from './snaps.js';
 
 const BASE_URL = conf.get('BASE_URL');
 
-export const FETCH_REPOSITORIES = 'FETCH_ALL_REPOSITORIES';
-export const FETCH_REPOSITORIES_ERROR = 'FETCH_REPOSITORIES_ERROR';
-export const SET_REPOSITORIES = 'SET_REPOSITORIES';
-export const SET_REPOSITORY_PAGE_LINKS = 'SET_REPOSITORY_PAGE_LINKS';
+export const REPOSITORIES_REQUEST = 'REPOSITORIES_REQUEST';
+export const REPOSITORIES_SUCCESS = 'REPOSITORIES_SUCCESS';
+export const REPOSITORIES_FAILURE = 'REPOSITORIES_FAILURE';
+
 
 export function setRepositories(repos) {
   return {
-    type: SET_REPOSITORIES,
```

Comment:
The action doesn't set repositories, the reducer does that, the action is some excitable teenager shouting "hey guys, we got our repositories list from github!" with no clue what to do with that list.